scripts/ddd/ddd

```diff
@@ -95,7 +95,11 @@ restore_mapping() {
 
 	# If we have stored USB_MAP, reload it
 	if [ -f ${DDD_MAP} ]; then
-		source ${DDD_MAP}
+        source ${DDD_MAP}
```

Comment:
Hmm, I think these end up doing the same thing, when we do the declare -p write, it writes all the values, and sourcing that file should end up doing the same thing. We can keep it like this now, since it's safer and pretty much the same thing :smile: